code-for-canada/project-thundercat
# [Add/Edit Task] Connection with Redux + Display added tasks

# Description

- I connected 'Add Task' action to Redux
- I introduced the task view using _CollapsingItemContainer.jsx_ component when task are added

## Type of change

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

**Creating a Task (1):**

![image](https://user-images.githubusercontent.com/23021242/55882502-a68a6d80-5b72-11e9-8b59-cdbb6d02b1a4.png)

**Creating a Task (2):**

![image](https://user-images.githubusercontent.com/23021242/55882653-e2bdce00-5b72-11e9-8207-f5345d48e574.png)

**Displaying added Tasks (1):**

![image](https://user-images.githubusercontent.com/23021242/55882717-ff5a0600-5b72-11e9-882d-28df4d428140.png)

**Displaying added Tasks (2):**

![image](https://user-images.githubusercontent.com/23021242/55882772-113ba900-5b73-11e9-9e4b-6285dece7c52.png)

**Adding Email Response:**

![image](https://user-images.githubusercontent.com/23021242/55882891-40eab100-5b73-11e9-9ea7-10ab332ccb32.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Complete the _pre-test_ process.
3.  Open the _Inbox_ tab.
4.  Add some emails responses and tasks and make sure that you can see them once added to an email.

# Checklist

Applicable for all code changes.
